shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -241,6 +268,35 @@ - (void)removeKeyResponder:(NSResponder*)responder {
 
 #pragma mark - Private methods
 
+- (void)configureTrackingArea {
+  if (_mouseTrackingMode != FlutterMouseTrackingModeNone && self.view) {
+    NSTrackingAreaOptions options =
+        NSTrackingMouseEnteredAndExited | NSTrackingMouseMoved | NSTrackingInVisibleRect;
+    switch (_mouseTrackingMode) {
+      case FlutterMouseTrackingModeInKeyWindow:
+        options |= NSTrackingActiveInKeyWindow;
+        break;
+      case FlutterMouseTrackingModeInActiveApp:
+        options |= NSTrackingActiveInActiveApp;
+        break;
+      case FlutterMouseTrackingModeAlways:
+        options |= NSTrackingActiveAlways;
+        break;
+      default:
+        NSLog(@"Error: Unrecognized mouse tracking mode: %ld", _mouseTrackingMode);
+        return;
+    }
+    _trackingArea = [[NSTrackingArea alloc] initWithRect:NSZeroRect
```

Comment:
This is ARC right?

Reply:
Yes, all the macos/framework code is compiled with ARC enabled.